### ISSUE TITLE
Add option to reuse workers in test suite

### ISSUE THREAD

**Issue by alanking** [irods/irods_capability_automated_ingest#126]
Each test case stands up Celery workers as part of setUp and then brings them down in tearDown. This allows for complete idempotency in the tests, but is also very slow and at times unnecessary. There should be some kind of option to allow for workers to be instantiated once and run multiple tests before bringing them down again.

**Comment by alanking** [comment]
This is kind of being done in the new test suite `test_delete_modes`. The old test suite is pretty brittle, so maybe we can just do this for any new tests moving forward instead of modifying the old ones...

**Comment by korydraughn** [comment]
Agreed. Limiting this to new tests will allow us to develop the approach and once we're confident, adjust the old tests if necessary.
